pkg/server/api/entry/v1/service.go

```diff
@@ -169,6 +175,83 @@ func (s *Service) BatchDeleteEntry(ctx context.Context, req *entry.BatchDeleteEn
 	}, nil
 }
 
+func (s *Service) updateEntry(ctx context.Context, e *types.Entry, inputMask *types.EntryMask, outputMask *types.EntryMask) *entry.BatchUpdateEntryResponse_Result {
+	log := rpccontext.Logger(ctx)
+
+	if e == nil {
+		log.Error("missing entry")
+		return &entry.BatchUpdateEntryResponse_Result{
+			Status: api.CreateStatus(codes.Internal, "missing entry"),
+		}
+	}
+
+	if inputMask == nil {
+		log.Error("missing input mask")
+		return &entry.BatchUpdateEntryResponse_Result{
+			Status: api.CreateStatus(codes.Internal, "missing input mask"),
+		}
+	}
+	if e.SpiffeId != nil {
```

Comment:
instead of adding  field for spiffeID we must use entryID that must not be nil

Reply:
Yeah that makes more sense... or both?